priv/skel/basesite/config.in

```diff
@@ -4,7 +4,7 @@
     {enabled, true},
 
     % Hostname on which this site runs
-    {hostname, "%%SITEHOSTNAME%%:8000"},
```

Comment:
That is part of what you type as hostname.

The listen port and the "outside" port are user configurable, so this shouldn't be in the skel file anyway.